pandas/_libs/src/parser/tokenizer.c

```diff
@@ -1784,6 +1784,8 @@ char* _str_copy_decimal_str_c(const char *s, char **endpos, char decimal,
     size_t length = strlen(s);
     char *s_copy = malloc(length + 1);
     char *dst = s_copy;
+    // Skip leading whitespace.
```

Comment:
is this routine only exercised for the high precision case or always? shouldn't skipping depend on the skipintialspace being true?

Reply:
> is this routine only exercised for the high precision case or always?

For the high precision ("round_trip") only.

> shouldn't skipping depend on the skipintialspace being true?

Probably but the float parsing functions are not parameterized with `skipintialspace`. In fact I think that `skipintialspace` has no effect on float parsing.